docs/guide/data_migration/exporting_and_importing_data.md

```diff
@@ -310,6 +310,41 @@ Create a currency:
     subunits: 3
 ```
 
+### Criteria
+
+When using `update` or `delete` modes, you can use criteria to identify the objects to operate on.
+
+``` yaml
+type: currency
+mode: update
+criteria:
+    type: field_value
+    field: code
+    value: EUR
+    operator: '=' # default
+code: EEE
+subunits: 3
+enabled: false
+```
+
+Available operators are: `=`, `IN`, `CONTAINS`, `STARTS_WITH` and `ENDS_WITH`.
```

Comment:
Available operators are actually defined in `Doctrine\Common\Collections\Expr\Comparison`.

```php
/**
 * Comparison of a field with a value by the given operator.
 */
class Comparison implements Expression
{
    public const EQ          = '=';
    public const NEQ         = '<>';
    public const LT          = '<';
    public const LTE         = '<=';
    public const GT          = '>';
    public const GTE         = '>=';
    public const IS          = '='; // no difference with EQ
    public const IN          = 'IN';
    public const NIN         = 'NIN';
    public const CONTAINS    = 'CONTAINS';
    public const MEMBER_OF   = 'MEMBER_OF';
    public const STARTS_WITH = 'STARTS_WITH';
    public const ENDS_WITH   = 'ENDS_WITH';
    // ...
}
```
Out of this list, `MEMBER_OF` is not supported and will throw an exception is tried.